lambdas/utils/lloyd_george_validator.py

```diff
@@ -177,12 +151,28 @@ def validate_with_pds_service(file_name_list: list[str], nhs_number: str):
     except (ValidationError, ClientError, ValueError) as e:
         logger.error(e)
         raise LGInvalidFilesException(e)
+
+
+def getting_patient_info_from_pds(nhs_number: str):
```

Comment:
maybe add the return type here

```suggestion
def getting_patient_info_from_pds(nhs_number: str) -> PatientDetails:
```